### ISSUE TITLE
[REQUEST-FEATURE] - Change current tab with animation

### ISSUE THREAD

**Issue by maclacerda** [Johnykutty/JKSteppedProgressBar#7]
Hello,

Thank you for your amazing library.

It's possible implement the change current tab index with animation?

Thanks

**Comment by nzayatz14** [comment]
+1

**Comment by sarbogast** [comment]
That would be awesome indeed: especially when progress is updated in realtime, animations would make it easier to see something is changing. Like the activated color moving along the progress bar, and the newly activated step magnifying or something like that

**Comment by jkmathew** [comment]
@sarbogast , I agree. I am trying to achieve it with current implementation. Also lil busy with current work.


PRs always welcome :)
